packages/imperative/src/utilities/index.ts

```diff
@@ -26,3 +26,4 @@ export * from "./src/doc/ISystemInfo";
 export * from "./src/diff/DiffUtils";
 export * from "./src/diff/doc/IDiffOptions";
 export * from "./src/diff/doc/IDiffNameOptions";
+export * from "./src/DeferredPromise"
```

Comment:
Missing semicolon on line 29:

```suggestion
export * from "./src/DeferredPromise";
```